lib/commands/find.js

```diff
@@ -2,7 +2,11 @@ import _ from 'lodash';
 import { errors } from 'appium-base-driver';
 
 
-let commands = {}, helpers = {}, extensions = {};
+let helpers = {}, extensions = {};
+
+helpers.doFindElementOrEls = async function (params) {
```

Comment:
It would be great to have a little comment on this method (and all the extracted-out methods) saying why they are there. Otherwise someone might come along later and move them back in, thinking they are just redundant functions!